winterfell/webapp/src/js/claimSelectFormsCtrl.js

```diff
@@ -1,13 +1,23 @@
 var app = angular.module('vetafiApp');
-app.controller('claimSelectFormsCtrl', ['$scope', 'claimService', 'formTemplateService', '$stateParams',
-  function($scope, claimService, formTemplateService, $stateParams) {
+app.controller('claimSelectFormsCtrl', ['$scope', 'claimService', '$stateParams',
+  function($scope, claimService, $stateParams) {
     $scope.claimId = $stateParams.claimId;
-    $scope.forms = formTemplateService;
+    $scope.allForms = [
+      {
+        title: 'VA-2392 Intent to File',
+        summary: 'This form is meant for blah blah something to hold your place in line.',
+        required: true
+      }
+    ];
+    var partitionForms = _.partition($scope.allForms, 'required');
```

Comment:
This logic counting the number of required forms should be encapsulated in a function because we might need to call it multiple times if forms are updated or as a callback